bin/pycbc_optimize_snr

```diff
@@ -179,14 +185,18 @@ parser.add_argument('--pso-c2', type=float, default=2.0,
 parser.add_argument('--pso-w', type=float, default=0.01,
                     help='Only relevant for --optimizer pso: '
                     'The hyperparameter w: the intertia parameter.')
-                    
 
+# Input checking
 args = parser.parse_args()
+if args.snr_threshold != 4:
+    raise NotImplementedError("Sorry, the SNR threshold option doesn't '
+                              "work yet")
```

Comment:
If you end up keeping the unused option, please use
```python
parser.error('Sorry, the SNR threshold option does not work yet')
```
instead of an exception.

Reply:
is that this? https://docs.python.org/3/library/argparse.html?highlight=parser%20error#argparse.ArgumentParser.error